pyproject.toml

```diff
@@ -117,6 +115,10 @@ version_files = [
 target-version = "py38"
 line-length = 88
 fix = true
+extend-include = ["*.ipynb"]
+exclude = [
+    "walkthroughs/nbconverted/*",
+]
```

Comment:
Would this still be necessary if the `.py` file within `walkthroughs/nbconverted` is removed?